File: Splitwise.App/ClientApp/src/app/group/group-data.ts
import { User } from "../user/user";
import { UserBalanceInfo } from "../user/user-balance-info";

export interface GroupData {
  GroupId: number,
  GroupName: string,
  GroupCategory: string,
  CreatedBy: string,
  CreatedById: number,
  GroupLink: string,
  SimplifyDebts: boolean,
  CreatedAt: string,
  Modified: string,
  Users: Array<UserBalanceInfo>

}

File: Splitwise.Core/GroupData.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Splitwise.Core
{
    public class GroupData
    {
        public int GroupId { get; set; }
        public string GroupName { get; set; }
        public string GroupCategory { get; set; }
        public string CreatedBy { get; set; }
        public int CreatedById { get; set; }
        public string GroupLink { get; set; }
        public bool SimplifyDebts { get; set; }
        public string CreatedAt { get; set; }
        public string Modified { get; set; }
        public List<UserBalanceInfo> Users { get; set; }
    }
}
